src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
We never target a specific date. When we judge we have enough changes or someone really want/need a release to publish his/her crate, we do it.